packages/babel-plugin/src/css-prop/__tests__/behaviour.test.tsx

```diff
@@ -105,10 +101,7 @@ describe('css prop behaviour', () => {
       const Component = ({ className, style }) => <div className={className} style={{ ...style, display: 'block' }} css={{ fontSize: 12, color: red, background }}>hello world</div>;
     `);
 
-    expect(actual).toInclude(`style={{...style,display:'block',\"--var-hash-test\":background}}`);
-    expect(actual).toInclude(
-      `.cc-hash-test{font-size:12px;color:red;background:var(--var-hash-test)}`
```

Comment:
Did we miss this assertion? Can't see it 🤔

Reply:
i removed because it was just noise - the test wants to assert the spreading and we have the other assertion covered in other tests